whylog/assistant/exceptions.py

```diff
@@ -41,3 +41,24 @@ def __init__(self, assistant_class_name):
 
     def __str__(self):
         return 'This whylog version do not handle %s. Please upgrade Whylog' % self.assistant_class_name
+
+
+class SpanConstructorParamsError(WhylogError):
+    def __init__(self, start, end):
+        super(SpanConstructorParamsError, self).__init__()
```

Comment:
now WhylogError has a default constructor, but if one day someone will define the constructor and make it do something important, I don't think he should search through the whole source code for children and grandchildren of WhylogError to check if they are calling the superclass constructor or not. It is better to do it right from the start.